### ISSUE TITLE
More succes with steam trader helper than SCMbot

### ISSUE THREAD

**Issue by kraitfps** [Maxx53/SCMBot#407]
I have more succes with chrome extension, i don't know what i'm doing wrong with scmbot but there was a 5-10 Items placed on market with my desired price scmbot couldn't buy nothing. It says that someone bought  it, I tested it even with 10 delay.

What i'm doing wrong ?

**Comment by Jentosss** [comment]
Bot is dead, dude

**Comment by firebug11** [comment]
I don't think this bot is anywhere near dead, as i am having successes with it and a guy wrote the other week he has botted high end knives consistently.

Play around with the settings for most optimal efficiency.

**Comment by Shameles** [comment]
could u tell me name of knife he bouhgt? just curious

**Comment by Jentosss** [comment]
Karambit Doppler I guess )))

**Comment by Alex1os** [comment]
Hello kraitfps, did u actually buy the faster version of the chrome trader helper or are u using the free one?

I used the trader helper myself about half a year long and I can asure u that scm bot is way faster than the free trader helper.

@Jentosss Karambit Doppler is way to easy to get, u don't even need to bot for it because it's not worth much keys.Kara Doppler spawns on the market about 10-15 times per day! The really rare knives only 1-4 times.

Lg Alex

**Comment by kraitfps** [comment]
Free version, with scmbot I manage to get only 1 item from let say 30-40, no matter how low my delay is set. With trader helper I get bet 1 and 3 items from 20.

**Comment by Alex1os** [comment]
interesting, many ppl who use the free version of trader helper get mostly errors and i think its a bit slow, if I'm right the option "fast" in trader helper free version is only about 5000ms -10000ms fast, which is extremly slow compared to scm bot

but the advanced version of trader helper seems to have the same features as scm bot maybe even more

Lg Alex
